src/follow_up/components/MultiForm/ProgressBar/index.js

```diff
@@ -0,0 +1,15 @@
+import ProgressBar from 'react-bootstrap/ProgressBar'
+
+import classes from './styles.module.scss'
+
+function FormProgress ({ currentStep, totalSteps }) {
+  const progressPercentage = () => {
+    return Math.round((currentStep * 100) / totalSteps)
+  }
+
+  return (
+    <ProgressBar now={progressPercentage()} className={classes.progress} />
+  )
```

Comment:
Lo mismo con respecto a usar una función. Esto podés reemplazarlo con un `useMemo`, dejando el resultado en una variable:
```suggestion
  const progressPercentage = useMemo(() => {
    return Math.round((currentStep * 100) / totalSteps)
  }, [currentStep, totalSteps])

  return (
    <ProgressBar now={progressPercentage} className={classes.progress} />
  )
```